src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/crafting/BrewingStandListener.java

```diff
@@ -52,4 +53,12 @@ public void onPreBrew(InventoryClickEvent e) {
         }
     }
 
+    @EventHandler
+    public void hopperOnBrew(InventoryMoveItemEvent e) {
+        if (e.getDestination().getType().equals(InventoryType.BREWING)) {
+            if (isUnallowed(SlimefunItem.getByItem(e.getItem()))) {
```

Comment:
```suggestion
            if (isUnallowed(e.getItem())) {
```

Reply:
![image](https://user-images.githubusercontent.com/71848626/96912952-acc80c80-14c0-11eb-8ce0-99f966e56773.png)


Throws NPE, when `isUnallowed` is called without. Included a null check in method `isUnallowed` of `SlimefunCraftingListener` interface and did a little refractoring in the above `hasUnallowedItems` method. Let me know if this is ok.